### PR TITLE
TransformAzureOpenAI Implementation

### PR DESCRIPTION
**Pull request by frank-suwen**
Make the shift from OpenAI to AzureOpenAI by updating corresponding config and server files. Create a example to demonstrate its usage.

### CODE REVIEW

File: uniflow/op/model/lm/model_server.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 from functools import partial
 from typing import Any, Dict, List, Optional
 
+import requests
```
Comment (CambioML) [inline uniflow/op/model/lm/model_server.py:13]: why are we using the requests instead of the AzureOpenAI client directly?

Comment (frank-suwen) [reply uniflow/op/model/lm/model_server.py]: just replaced requests with AzureOpenAI client

File: example/transform/azure_openai_html_QA.ipynb.ipynb [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.13"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (CambioML) [inline example/transform/azure_openai_html_QA.ipynb.ipynb:445]: Can you add the `End of Notebook` section to the end here (copy from this notebook example: https://github.com/CambioML/uniflow-llm-based-pdf-extraction-text-cleaning-data-clustering/blob/8e0a48abb5a20746ea0ed0b7cc6b6c99f529a379/example/transform/huggingface_model_benchmark_g5.ipynb)

**Review by CambioML — CHANGES_REQUESTED** [review]
As mentioned in the comment, please add the standard example notebook ending

**Review by CambioML — DISMISSED** [review]
Please resolve merge conflicts with main

**Review by CambioML — APPROVED** [review]
LGTM!

**Pull request merged**